fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -1099,7 +1097,6 @@ private List<Partition> createPartitionList(Database db, OlapTable copiedTable,
                     createPartition(db, copiedTable, partitionId, partitionName, version, tabletIdSet);
 
             partitionList.add(partition);
-            tabletIdSetForAll.addAll(tabletIdSet);
```

Comment:
`tabletIdSetForAll` is used to delete tablet from inverted index when create partition failed.